railties/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Conditionally skip test job in ci.yml
+
+    Skips the `test` job when a new application is generated with the
+    `--skip-test` option.
```

Comment:
```suggestion
*   Skip generating a `test` job in ci.yml when a new application is generated with the
    `--skip-test` option.
```

This may be personal preference, but I didn't feel like the summary added much value vs. just putting the whole thing in one line.